modules/client-configuration/pages/client-operations.adoc

```diff
@@ -0,0 +1,10 @@
+[[client.operations]]
+= Client Operations
+
+In addition to registering, upgrading or deleting clients, a number of other operations can be performed.
```

Comment:
```suggestion
In addition to registering, upgrading, or deleting clients other operations can be performed.
```
Or maybe, just:

> Registering, upgrading, or deleting clients and other operations can be performed.

(But I do not want to judge about the style we want to use.)